src/navigator.js

```diff
@@ -18,6 +20,46 @@ Handlebars.registerHelper('encodeURI', function(uri){
     return encodeURI(uri);
 });
 
+Handlebars.registerHelper('metricsBarGraph', function(nodes, fn){
```

Comment:
I think it would be better to pass the data variable to the template and allow the template to call toJSON (defined above) inside a script tag in the template. This would avoid this code generation (js that creates js) and would make this charting code easier to maintain.